recipes/pdal/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -ex
+
+if [ $(uname) == Darwin ]; then
+    export CXXFLAGS="-stdlib=libc++ $CXXFLAGS"
+    CC=clang
+    CXX=clang++
+    export MACOSX_DEPLOYMENT_TARGET="10.9"
+else
+    export CXXFLAGS="$CXXFLAGS -std=c++11"
+fi
+
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DCMAKE_LIBRARY_PATH=$PREFIX/lib \
+  -DCMAKE_INCLUDE_PATH=$PREFIX/include \
+  -DBUILD_PLUGIN_PYTHON=ON \
+  -DBUILD_PLUGIN_PGPOINTCLOUD=ON \
+  -DBUILD_PLUGIN_SQLITE=ON \
+  -DBUILD_PLUGIN_ICEBRIDGE=ON \
+  -DENABLE_CTEST=OFF \
+  -DWITH_TESTS=OFF \
+  -DWITH_LAZPERF=OFF \
+  -DWITH_LASZIP=OFF
+
+# CircleCI offers two cores.
+make -j $CPU_COUNT
+make install
+
+cd python
+python setup.py build
```

Comment:
We've split `gdal` into `gdal` (python) and `libgdal` (C++). Do we want to do the same here?

Reply:
Is it a convention of conda-forge that we need to adapt to? If so, I can update. Otherwise, I'm not sure I see the advantage.

Reply:
Our alpine packages come in three flavors: `py-pdal`, `pdal`, and `pdal-dev`. I like that convention, but perhaps because I'm familiar with it. Conda is still relatively new to me.

Following the model you suggest, `pdal` would install the python bindings, and `libpdal` would install both the libraries and the `pdal` application itself. I feel like users could be tripped up by that -- they may just want the `pdal` application and think that they need to install the `pdal` conda package. It wouldn't really be an issue I suppose, as `libpdal` would be a runtime requirement of the python bindings and would be installed regardless. It just seems a little confusing.

Reply:
`gdal` was initially C++ and Python bindings together, but when we wanted to split it, we couldn't go with `py-gdal` and `gdal` for backwards compatibility reasons. That's not the case here, so both are valid naming schemes to me.

@ocefpaf, any suggestions on the naming?

Reply:
Btw, from what you said about the alpine packages, I assume that you are in favor of splitting the packages into shared library and python package. Only issue is the naming, right?

Reply:
> Is it a convention of conda-forge that we need to adapt to? If so, I can update. Otherwise, I'm not sure I see the advantage.

No. We did that to adapt with `defaults`. I'm OK with a single package here if that is more convenient.

Reply:
@isuruf I'll split into `py-pdal` and `pdal` so that we don't get into a bind like `gdal` did. Is it possible to submit both recipes in a single PR, as `py-pdal` will depend on `pdal`?

Reply:
sure. `py-pdal-feedstock`  would have to be rerendered later using `conda smithy rerender` if both are submitted in a single PR

Reply:
In that case I'd rather if you name it `python-pdal` and `pdal` (or maybe `libpdal`?) to follow the rest of `conda-forge` naming scheme.

You can send both in a single PR but then you'll get a race condition during the feedstock creation that will require us to re-start the CIs on the package that depends on the other. (That is not a big deal.)